src/inputs.ts

```diff
@@ -1,6 +1,7 @@
-import { DirectomaticResponse, RawRedirectProps } from '.';
+import { DirectomaticResponse, RawRedirectProps } from './types';
 
-const lookup = `${GSHEETS_API_ENDPOINT}/${GSHEETS_ID}/values/Redirects!A:E?key=${GSHEETS_API_KEY}&valueRenderOption=UNFORMATTED_VALUE`;
+const lookup = (): string =>
```

Comment:
Had to make this a function so we can setup the environment correctly in the unit tests.